### ISSUE TITLE
Q2.1 and Q2.2: First touch and last touch.  Add one line answer to the question. 

### ISSUE THREAD

**Issue by siyuan126** [kaps3002/sql-scratch-capstone-turn-in#3]
Similarly. Please add one line answer to indicate how many first touches and last touches.

**Comment by kaps3002** [comment]
Again as I had explained earlier all the answers are in the table format. Powerpoint slides are more for graphical representation of data rather than endless notes on each slide.

**Comment by siyuan126** [comment]
Hi, for this question, it asks for how many first touches and last touches. The answer we are looking for is very simple. 
**There were 1,979 first touches.** People found the site mostly through article campaigns:
interview-with-cool-tshirts-founder (medium)
getting-to-know-cool-tshirts (nytimes)
ten-crazy-cool-tshirts-facts (buzzfeed)

And as comparison, here is the answer you provide.
<img width="483" alt="screen shot 2018-08-11 at 3 47 35 pm" src="https://user-images.githubusercontent.com/10052104/43995568-07944578-9d7e-11e8-9496-ee9e4b98b3f7.png">

I totally agree that ppt should only include most important info. Here the most important info is the **answer** and **evidence**. Not thinking process or background info.

**Comment by kaps3002** [comment]
The capstone project is specifically asking for How many first touches and last touches is **each campaign responsible for?**

If the question was how many total first touches and last touches I would have answered that but as you can see from the below screenshot (taken from the project page) you're only interested in numbers for each campaign.

![image](https://user-images.githubusercontent.com/7575095/43995624-663ca90c-9d7f-11e8-8696-fcd3ba37c137.png)

**Comment by siyuan126** [comment]
Yes. It is true that it asks for the number of first touches from each campaign. And I see your table. So what are the conclusions from the table? 

<img width="576" alt="screen shot 2018-08-11 at 4 09 33 pm" src="https://user-images.githubusercontent.com/10052104/43995700-fa3112c8-9d80-11e8-9ed3-de62b01c44a7.png">

A summary of the findings is what we are looking for.  Are these of the conclusions you have from this table? 
<img width="483" alt="screen shot 2018-08-11 at 3 47 35 pm" src="https://user-images.githubusercontent.com/10052104/43995715-2921337e-9d81-11e8-809d-0b0d0967502e.png">

**Comment by kaps3002** [comment]
My conclusions are provided in the slides labelled 2.5 User Journey. This is where I analyze the data and give my conclusions.

![image](https://user-images.githubusercontent.com/7575095/43995764-a0777a86-9d82-11e8-8d7b-444ed89706d9.png)

**Comment by siyuan126** [comment]
These are questions 2.1 and 2.2. And answers are expected to be seen **clearly here**, not in other different questions. A very simple way to answer this question is something like this: 
**How many first touches is each campaign responsible for?** 
interview-with-cool-tshirts-founder (medium) generate 622 first touches
getting-to-know-cool-tshirts (nytimes) generate 612 first touches
ten-crazy-cool-tshirts-facts (buzzfeed) generate 576 first touches 
google-search generates 169 touches. 
(this version is wordy but can be a good start)

Or you can answer this question like this:
There were 1,979 first touches. People found the site mostly through article campaigns:
interview-with-cool-tshirts-founder (medium)
getting-to-know-cool-tshirts (nytimes)
ten-crazy-cool-tshirts-facts (buzzfeed)

You have the table, just **missing the one line answer to the question**. Imagine someone ask you this question, can you just say " oh, here is the table, you can look at yourself"? Or you would rather say (just an example): " actually three article campaigns generate about 1,800 first touches, and google search is not very effective, only 190 first touches. The latter is what we are looking for.

**Comment by kaps3002** [comment]
I do feel that the questions for first and last touches (2.1 & 2.2) are not asking me to draw conclusions from the results. Only in the latter part of the project is where I've been asked some analysis related questions, like drawing conclusions. And instead of answering them in a verbose manner I'm showing them in a pie chart and presenting the percentages instead of the totals.

Well at this point, I'm just gonna stop debating here. Thank you for your time.

**Comment by siyuan126** [comment]
it just asks you questions and answers needed to be provided. One line answers are missing here. This is the point, for clarity.
